loky/__init__.py

```diff
@@ -10,9 +10,12 @@
 
 from .backend.context import cpu_count
 from .backend.reduction import set_loky_pickler
-from .reusable_executor import get_reusable_executor
+try:
+    from .reusable_executor import get_reusable_executor
+    from .process_executor import BrokenProcessPool, ProcessPoolExecutor
+except ImportError:
```

Comment:
Please add a comment to explain why:

```suggestion
except ImportError:
    # On environments like pyodide that do not implement the _multiprocessing
    # module in the standard library, we still want loky to be importable without
    # failing, even if not really usable.
```